grails-app/services/com/mini/asaas/payer/PayerService.groovy

```diff
@@ -29,4 +29,10 @@ class PayerService {
         
         return payer
     }
+
+    public List<Payer> findAllByCustomer(Long customerId) {
+        Customer customer = Customer.read(customerId)
+
+        return Payer.findAllByCustomer(customer)
```

Comment:
E se tiver um Payer com soft delete? Também deve ser retornado aqui?

Reply:
Fiz essa alteração na branch que vai adicionar soft delete de pagadores, mas se for melhor colocar aqui, posso fazer isso.